.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseCJKNumberParser.cs

```diff
@@ -548,7 +550,7 @@ private double GetIntValue(string intStr)
                 {
                     if (i != intStr.Length - 1)
                     {
-                        if (intStr[i] == '零' && !Config.RoundNumberMapChar.ContainsKey(intStr[i + 1]))
+                        if (intStr[i] == roundNumberZero && (!Config.RoundNumberMapChar.ContainsKey(intStr[i + 1]) || (Config.CultureInfo.Name == "zh-CN" && System.Array.IndexOf(roundNumberTen, intStr[i + 1]) != -1)))
```

Comment:
This if could be a call to a method.
Also, it should not reference a culture directly. Whatever culture-specific behaviour is needed should go into the Config class for that culture. Either as attributes or methods.